packages/sarif-to-comment/test/empty-comment.test.ts

```diff
@@ -0,0 +1,25 @@
+import * as fs from "fs";
+import * as path from "path";
+import * as assert from "assert";
+// transform function
+import { postComment } from "../src/index";
+
+console.log(__dirname);
+const sarifDir = path.join(__dirname);
+const actualFilePath = path.join(sarifDir, "/sarif.json");
+describe("Testing", () => {
+    it(`Testing failure due to empty sarif`, async function () {
+        const actualOptions = {
+            sarifContent: JSON.parse(fs.readFileSync(actualFilePath).toString()),
```

Comment:
```suggestion
            sarifContent: JSON.parse(fs.readFileSync(actualFilePath, "utf-8")),
```

Buffer.toString is unnecessary expense.

Reply:
the problem is that I get an error if I didn't use toString()`Argument of type 'Buffer' is not assignable to parameter of type 'string'.ts(2345)`